DEVELOPER_GUIDE.md

```diff
@@ -53,6 +50,15 @@ opensearch.password: "admin" # Default password on the docker image
 opensearch.ssl.verificationMode: none
 ```
 
+### Run Opensearch
+
+You need to have an OpenSearch server up and running to be able to run OpenSearch
+Dashboards. In a separate terminal you can run the latest nightly build using
```

Comment:
NIT: I think you should be:

`In a separate terminal you can run the latest snapshot built using:`

I think the plan is to have snapshots build every 2 hours if there is a change. But it is subject to change.